### ISSUE TITLE
I would love if I could snoze one quarter and one year 

### ISSUE THREAD

**Issue by paven** [bwinton/SnoozeTabs#483]
for sites, I would like to repeat again later for learning reputation, but not urgent.

**Comment by todaysmyfirstday** [comment]
This is a feature in "Snooze this Tab" in Chrome.  It would be great to see this on Firefox, as well as being able to set a time for "Later Today" which I think most 9-5ers want to be after work hours.

**Comment by bwinton** [comment]
Would you want it to snooze for 3 months, or until the start of the next quarter?
And what about for the year option…

**Comment by paven** [comment]
3 months(~same day of month) as I would prefer to have thing's portioned out. 
Years 365 days forward same reason.

**Comment by todaysmyfirstday** [comment]
I agree with Patrick. Also, the later today option would be better if it
was after 5pm, since sometimes o want to revisit things after my workday.

**Comment by bwinton** [comment]
That all makes sense. I think my only concern at this point is that the number of options is getting a little unwieldy… (Particularly if I also add a "snooze for 5 minutes" option, as another person asked for.)

**Comment by todaysmyfirstday** [comment]
True, although I don't see the point of a snooze for 5 minutes.  If it is
only that long, just leave it open.

**Comment by bwinton** [comment]
https://github.com/bwinton/SnoozeTabs/issues/448 😄

**Comment by sNiXx** [comment]
But isn't #448 asking for a different kind of option? As far as I understand, the "quick snooze" would work like it does on Android (i.e. a notification that you use to dismiss or snooze). The quarter and year option would rather be alongside the other options, such as "Later Today" and "Tomorrow" in the extension drop-down itself?

Maybe to address Patrik's request, and keeping the number of options small, you could add an option to snooze for X days. X is then either defined in the settings or when clicked?

PS: cool that you are again working on this fantastic extension, @bwinton!

**Comment by paven** [comment]
There are snooze options to pick dates and stuff. That is not what I am asking for. I want the UX to be quick for long snoozes to. When i set up similar systems i snoozed for 1 hour, 1 day, 1 week, 1 month, 1 quarter, 1 year.  For me there need no 1 minute option, as i would just keep the tab open. 1 hour is not so useful either, but i used it from time to time.

**Comment by paven** [comment]
Perhaps, settings page, where options can be picked to be displayed?
with default settings to whatever is most commonly used.

**Comment by todaysmyfirstday** [comment]
Yes, I'll echo SNixX, very cool that you're working on this incredibly
useful feature.  I think that the # of options could become too large.  I
had a similar extension when I used Chrome, that would allow for a tab to
open on a schedule (ie: 8am every day) that was very useful.  I am not sure
that 1 year from now needs to be a "quick pick" option in my usage.  I
think "Repeat on Schedule", "End of Business Day", "Weekend", are the three
most useful quick options.  1 month from now shows up in the calendar (one
extra mouse click), and there aren't too many things to put off for a
year.  Similarly, if you can't ignore a tab for 5 minutes, then you have
bigger problems.  Personally, I use the extension to delay personal items
until I have more personal time to deal with them.  Once it's into that
personal time, then I can take more time and select when I want it to
reappear (3m, 12m, etc) and can select the time from the calendar.
